dom.bs

```diff
@@ -958,48 +958,48 @@ are not to be used for anything else. [[!HTML]]
 
 <dl class=domintro>
  <dt><code><var>target</var> = new <a constructor for=EventTarget lt=EventTarget()>EventTarget</a>();</code>
- <dd>
-  Creates a new {{EventTarget}} object, which can be used by developers to <a>dispatch</a> and listen for
-  <a>events</a>.
+ <dd><p>Creates a new {{EventTarget}} object, which can be used by developers to <a>dispatch</a> and
+ listen for <a>events</a>.
 
  <dt><code><var>target</var> . <a method for=EventTarget lt=addEventListener()>addEventListener</a>(<var>type</var>, <var>callback</var> [, <var>options</var>])</code>
  <dd>
-  Appends an <a>event listener</a> for <a>events</a> whose {{Event/type}} attribute value
-  is <var>type</var>. The <var>callback</var> argument sets the <b>callback</b> that will
-  be invoked when the <a>event</a> is <a>dispatched</a>.
+  <p>Appends an <a>event listener</a> for <a>events</a> whose {{Event/type}} attribute value is
+  <var>type</var>. The <var>callback</var> argument sets the <a for="event listener">callback</a>
+  that will be invoked when the <a>event</a> is <a>dispatched</a>.
 
-  The <var>options</var> argument sets listener-specific options. For compatibility this can be just
-  a boolean, in which case the method behaves exactly as if the value was specified as
+  <p>The <var>options</var> argument sets listener-specific options. For compatibility this can be a
+  boolean, in which case the method behaves exactly as if the value was specified as
   <var>options</var>' <code>capture</code> member.
```

Comment:
While here you could make all these `<code>`s link to the dictionary member definitions, as is done in the algorithms below.